src/app/game-state/character.ts

```diff
@@ -282,6 +282,9 @@ export class Character {
     if (this.money > this.maxMoney){
       this.money = this.maxMoney;
     }
+    if (this.money > this.maxMoney){
```

Comment:
Not sure why this code chunk is duplicated here.

Reply:
No idea either, probably a bad merge somewhere.